### ISSUE TITLE
New rule: meta element has no refresh delay

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#495]
The ACT-Rules community group has completed work on `meta element has no refresh delay`. We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: meta element has no refresh delay
**Rule id**: bc659a
**Link to the rule**:https://act-rules.github.io/rules/bc659a#acknowledgments
**License**: W3C Software And Document Notice And License
**Implementors**:
- SortSite
- Alfa
- QualWeb

If you'd like to contact me, please e-mail me at wilco.fiers@deque.com. You can also reach me on the W3C Community Slack, my name there is Wilco Fiers.

**Comment by maryjom** [comment]
[Survey]() due 3 December due to Thanksgiving holiday.

**Comment by maryjom** [comment]
See [issue 505](https://github.com/w3c/wcag-act/issues/505) documenting survey comments.
